Makefile

```diff
@@ -7,10 +7,18 @@ GOTEST=$(GOCMD) test
 AMKO_BIN=amko
 FEDERATOR_BIN=amko-federator
 SERVICE_DISCOVERY_BIN=amko-service-discovery
+PACKAGE_PATH=github.com/vmware/global-load-balancing-services-for-kubernetes
 AMKO_REL_PATH=github.com/vmware/global-load-balancing-services-for-kubernetes/cmd/gslb
 FEDERATOR_REL_PATH=github.com/vmware/global-load-balancing-services-for-kubernetes/federator
 SERVICE_DISCOVERY_REL_PATH=github.com/vmware/global-load-balancing-services-for-kubernetes/cmd/service_discovery
 
+
+ifdef GOLANG_SRC_REPO
+	BUILD_GO_IMG=$(GOLANG_SRC_REPO)
+else
+	BUILD_GO_IMG=golang:latest
+endif
+
 .PHONY: all
 all: vendor build
 
```

Comment:
build changes are missing. please refer this PR [https://github.com/vmware/global-load-balancing-services-for-kubernetes/pull/176/files#diff-76ed074a9305c04054cdebb9[…]07091de1f20cad0bbac34ffb52R26](https://github.com/vmware/global-load-balancing-services-for-kubernetes/pull/176/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R26)

Reply:
this is taken care now.